stable/nginx-ingress/values.yaml

```diff
@@ -8,6 +8,11 @@ controller:
     tag: "0.19.0"
     pullPolicy: IfNotPresent
 
+  configMap:
+    ## point to an existing configMap rather than creating one
+    name: ""
+
+  # config adds additional configurations to the created configMap when configMap.create=true
```

Comment:
`# config adds additional configurations to the created configMap unless an alternative configMap is set via controller.configMap`